backend/test/services/organization/organization_demo_data.py

```diff
@@ -268,7 +268,7 @@
     public = False,
 )
 
-organizations = [cads, cssg, appteam, acm, bit, arvr, ctf, enablingtech, esports, gamedev, gwc, hacknc, ktp, pearlhacks, pm, queerhack, wics]
+organizations = [appteam, acm, bit, cads, carvr, cssg, ctf, enablingtech, esports, gamedev, gwc, hacknc, ktp, pearlhacks, pm, queerhack, wics]
```

Comment:
Nit: Looks good! Thanks for catching this -- I think this should be a functionality of the service, however, in the future! Data will not necessarily get added regularly, however we will not be able to manually sort production data upon each post request. A sort pipe will probably work in the frontend for this. It would be a design choice on whether we want the API to return a sorted result or not. I am thinking this can be reserved for the front end but we can discuss this @KrisJordan  / @briannata  / @jadekeegan.